addons/mail/wizard/mail_compose_message_views.xml

```diff
@@ -70,8 +69,7 @@
                         </page>
                     </notebook>
                     <footer>
-                        <button string="Send" attrs="{'invisible': [('is_log', '=', True)]}" name="action_send_mail" type="object" class="btn-primary o_mail_send" data-hotkey="q"/>
-                        <button string="Log" attrs="{'invisible': [('is_log', '=', False)]}" name="action_send_mail" type="object" class="btn-primary" data-hotkey="q"/>
+                        <button string="Send" name="action_send_mail" type="object" class="btn-primary o_mail_send" data-hotkey="q"/>
```

Comment:
Should we provide some way so send the composer with the mt_note?

Or do we consider it should come from default values configuration and here you don't have the choice anymore?